diff --git a/dist/rehype-pretty-code.js b/dist/rehype-pretty-code.js
index cb97accbe303c1d75d5f4f8f6c607cb92ba37db8..5d455801111295de6c197678c3ec3cc5850ad545 100644
--- a/dist/rehype-pretty-code.js
+++ b/dist/rehype-pretty-code.js
@@ -12815,7 +12815,12 @@ function rehypePrettyCode(options = {}) {
         }
 
         // TODO: allow escape characters to break out of highlighting
-        const strippedValue = value.replace(/{:[a-zA-Z.-]+}/, '');
+        const strippedValue = codeNode.value.replace(/\n$/, '')
+                    .replaceAll(/&lt;/g, '<')
+                    .replaceAll(/&gt;/g, '>')
+                    .replaceAll(/&#123;/g, '{')
+                    .replaceAll(/&#125;/g, '}')
+                    .replaceAll(/&amp;/g, '&');
         const meta = value.match(/{:([a-zA-Z.-]+)}$/)?.[1];
 
         if (!meta) {
